pkg/cliplugins/bind/plugin/bind.go

```diff
@@ -44,7 +44,7 @@ type BindOptions struct {
 	// APIExportRef is the argument accepted by the command. It contains the
 	// reference to where APIExport exists. For ex: <absolute_ref_to_workspace>:<apiexport>.
 	APIExportRef string
-	// Name of the APIBinding.
+	// Path of the APIBinding.
```

Comment:
Should this be name or path? The field name is APIBindingName, but the comment talks about path.

Reply:
path. fixed.

Reply:
name